Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocListCssClassDefinition.cs

```diff
@@ -43,7 +44,20 @@ public virtual string Base
     /// </remarks>
     public virtual string ReadOnly
     {
-      get { return "readOnly"; }
+      get { return CssClassDefinition.ReadOnly; }
+    }
+
+    /// <summary>
+    /// Gets the CSS-Class applied to the <see cref="IBocRenderableControl"/> when itself and child elements
+    /// that are standard browser controls (e.g. input elements) should be styled in the current theme.
+    /// </summary>
+    /// <remarks> 
+    ///   <para> Class: <c>remotion-themed</c>. </para>
+    ///   <para> Applied in addition to the regular CSS-Class.</para>
+    /// </remarks>
+    public virtual string RemotionThemed
```

Comment:
You did resolve this conversation here but didn't reason pro/con for keeping `RemtionThemed` instead of `Themed` (or a better name).
https://github.com/re-motion/Framework/pull/162#discussion_r681532806
There may be a good reason, I just don't know it yet :)

Oh, okay, in https://github.com/re-motion/Framework/pull/162/files#diff-06753ed379cbf05cb3a5ae992083845b2e1e11c71db221e4d5799736c1aa9dc2 it's already `Themed` instead of `RemotionThemed`, so it's just a matter of making it consistent now?

Reply:
Jup i forgot to change it here apparently 😅